src/SFML/Graphics/Font.cpp

```diff
@@ -481,11 +495,19 @@ float Font::getUnderlineThickness(unsigned int characterSize) const
     }
 }
 
-
 ////////////////////////////////////////////////////////////
-const Texture& Font::getTexture(unsigned int characterSize) const
+std::unordered_set<std::uint64_t> Font::getTextureIds(unsigned int characterSize) const
 {
-    return loadPage(characterSize).texture;
+    PageList& pages = m_pageLists[characterSize];
+    if (pages.empty())
+        pages.push_back(Page{m_isSmooth});
```

Comment:
Use constructor syntax `Page(m_isSmooth)`.

Reply:
Why?

Reply:
Actually can just replace with emplace_back then it doesn't matter

Reply:
> Why?

Because that's the convention SFML uses and there is no reason to deviate in this instance.